tests/sentry/search/test_utils.py

```diff
@@ -514,3 +515,17 @@ def test(self):
             environment = self.create_environment()
             result = get_latest_release([self.project], [environment])
             assert result == new.version
+
+
+class ConvertUserTagTest(TestCase):
+    def test_simple_user_tag(self):
+        assert convert_user_tag_to_query("user", "id:123456") == 'user.id:"123456"'
+
+    def test_user_tag_with_quote(self):
+        assert convert_user_tag_to_query("user", 'id:123"456') == 'user.id:"123\\"456"'
+
+    def test_user_tag_with_space(self):
+        assert convert_user_tag_to_query("user", "id:123 456") == 'user.id:"123 456"'
```

Comment:
black is likely going to get salty about the single quoted strings.

Reply:
So Black flipped me to the single quoted strings cause originally I had
`"user.id:\"123 456\""`

Reply:
lololol.